src/index.spec.js

```diff
@@ -131,7 +131,7 @@ describe('Mongo', () => {
 
   describe('findUserById', () => {
     it('should return null for not found user', async () => {
-      const ret = await mongo.findUserById('unknowuser');
+      const ret = await mongo.findUserById('589871d1c9393d445745a57c');
```

Comment:
The Accounts server package will not check if the id is a valid ObjectId, I am not sure we want to throw this error if the user is not found rather than just return null.